openhtf/output/web_gui/__init__.py

```diff
@@ -387,13 +387,53 @@ def on_unsubscribe(self):
     self.subscriber_to_hostport_map.pop(self)
 
 
+class TestHandler(tornado.web.RequestHandler):
```

Comment:
It's fine. It looks like you have merged two handlers into this one, a `TestPhasesHandler` and a `TestHistoryHandler`. Can you separate them, either by creating a base class or a function to handle the common stuff in `get()`